luigi/file.py

```diff
@@ -43,8 +49,55 @@ def move_to_final_destination(self):
     def generate_tmp_path(self, path):
         return path + '-luigi-tmp-%09d' % random.randrange(0, 1e10)
 
+class atomic_file_append(atomic_file):
+    """Use this to append to a temporary file and append to the final
```

Comment:
No, it is correct. It will append, that is the purpose of it to append to the final destination.